### DISCUSSION TITLE
Convert Arabic (all Unicode range, specifically for Kurdish) PDF to txt

### DISCUSSION THREAD

**Discussion by 0xdolan** [diegomura/react-pdf#2271] [Ideas]
Does this software correctly render and convert Arabic (all Arabic Unicode range) text from PDF to text? or simply making it up? I believe it would be a good idea to include these things as well if there isn't already. I suggest to add support the entire Arabic range to convert all Arabic, Persian, and Kurdish writings correctly. I will be available to help with both Persian and Kurdish in the testing process (specifically Kurdish as it needs a lot of work).
Thank you!

**Comment by dyary98** [comment]
any updates?

**Comment by ruhussain** [comment]
Yes @0xdolan its better if this library support Arabic, Persian and Kurdish. Now, i am getting issue in arabic language which is not rendering on mac.
